saleor/graphql/site/mutations.py

```diff
@@ -0,0 +1,55 @@
+
+import graphene
+
+from ...site import models
+from ..core.mutations import BaseMutation
+from ..core.types.common import WeightUnitsEnum
+from .types import SiteSettings
+
+
+class SiteSettingsInput(graphene.InputObjectType):
+    domain = graphene.String(description='Domain name for shop')
+    header_text = graphene.String(description='Header text')
+    description = graphene.String(description='SEO description')
+    top_menu = graphene.ID(
+        description='Menu to use on site top')
+    bottom_menu = graphene.ID(
+        description='Menu to use on site bottom')
+    include_taxes_in_prices = graphene.Boolean(
+        description='Include taxes in prices')
+    display_gross_prices = graphene.Boolean(
+        description='Display prices with tax in store')
+    track_inventory_by_default = graphene.Boolean(
+        description='Enable inventory tracking')
+    homepage_collection = graphene.ID(
```

Comment:
Maybe this should be done in a dedicated mutation?

Reply:
Maybe as a rule of thumb, we should only put here simple settings such as booleans or strings, while having dedicated mutations for operations that manipulate objects or relationships?